### PR TITLE
React examples & docs

### PR DESCRIPTION
**Pull request by Comandeer**
I've added React guide and examples. It required adding several dependencies and new build step.

### CODE REVIEW

File: docs/sdk/examples/react.html [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+		config={{
+			toolbar: [
+				[ 'Bold', 'Italic' ],
+				[ 'Cut', 'Copy' ],
+				[ 'About' ]
+			]
+		}}
+		onFocus={customHandler}
+		onBlur={customHandler}
+		onChage={customHandler}
```
Comment (mlewand) [inline docs/sdk/examples/react.html]: Change events are not logged, there's a typo in this line, Chage -> Change.

File: docs/sdk/examples/react.html [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+	/>,
+	document.getElementById( 'app' )
+);
+
+function customHandler( evt ) {
+	console.log( evt );
+}
+</script>
+<script type="text/jsx" data-sample="4" data-sample-preserve-whitespace data-sample-strip-outer-tag data-sample-template="empty" data-sample-highlighter="jsx">
+import React from 'react';
```
Comment (mlewand) [inline docs/sdk/examples/react.html]: This sample will fail to compile, Component class is not defined.

Should be `import React, { Component } from 'react';`.

File: react/TwoWayBinding.jsx [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+		if ( !this.state.focused ) {
+			textareaValue = {
+				value: this.props.data
+			};
+		}
+
+		return (
+			<>
+				<p>
+					<label for="editor-editor">Change value:</label>
```
Comment (mlewand) [inline react/TwoWayBinding.jsx]: React suggest me using `htmlFor` instead of `for` 🙂

File: docs/sdk/examples/react.html [hunk 4]
```diff
@@ -0,0 +1,319 @@
...
+		if ( !this.state.focused ) {
+			textareaValue = {
+				value: this.props.data
+			};
+		}
+
+		return (
+			<>
+				<p>
+					<label for="editor-editor">Change value:</label>
```
Comment (mlewand) [inline docs/sdk/examples/react.html]: React suggest me using `htmlFor` instead of `for` 🙂

File: gruntfile.js [hunk 5]
```diff
@@ -6,14 +6,14 @@
...
 
 const Umberto = require( 'umberto' );
+const webpackReactConf = require( './react/webpack.config.js' );
 
 module.exports = function( grunt ) {
 	const packageVersion = grunt.file.readJSON( 'package.json' ).version;
 
 	require( 'load-grunt-tasks' )( grunt );
-
-	grunt.loadNpmTasks( 'grunt-contrib-connect' );
```
Comment (mlewand) [inline gruntfile.js]: Why did you remove this task? It will break the `docs-serve` task.

Comment (Comandeer) [reply gruntfile.js]: It shouldn't as this task is listed in dev deps and `load-grunt-tasks` loads tasks from all dependencies.
